src/Executor.cpp

```diff
@@ -2,17 +2,19 @@
 
 #include <folly/Memory.h>
 #include <folly/futures/QueuedImmediateExecutor.h>
+#include <folly/io/IOBuf.h>
 #include "src/StackTraceUtils.h"
 #include "src/SubscriberBase.h"
 
 namespace reactivesocket {
 
 // just instantiating of the template here
 template class SubscriberBaseT<Payload>;
+template class SubscriberBaseT<folly::IOBuf>;
```

Comment:
the explicit instantiation is related only to SubscriberBaseT<Payload> and SubscriberBaseT<folly::IOBuf>. For any other template param type you would get the normal instantiation